manuscript/glossary.md

```diff
@@ -0,0 +1,80 @@
+### Glossary and Abbreviations used in the manuscript
+
+#### Projects
+
+* **GBIF** (Global Biodiversity Information Facility, http://www.gbif.org) provides
+  an international open data infrastructure to publish and disseminate
+  biodiversity information.
+
+* **iDigBio** (Integrated Digitized Biocollections, https://www.idigbio.org) is a
+  project funded by the National Science Foundation that facilitates the
+  digitization of natural history collections, and provides data and images for
+  biological specimens.
+
+#### File Formats
+
+* **CSV** (Comma-Separated Values) and **TSV** (Tab-Separated Values) are plain text
+  file formats used to store tabular data where each row is represented by a line
+  in the file, and each field (column) is separated by a comma for CSV or by the
+  Tab character for TSV.
```

Comment:
I suspect readers who are unfamiliar with the terms TSV/CSV would better understand `separated by a comma for CSV or a tab for TSV.

On the lines of audience, would it be sufficient to discuss CSV and omit reference to TSV? Are there any contexts in which TSV is preferred to CSV, or could we just focus on 'use csv'? These comments refer to this part of the glossary and I haven'g reference the main text ... but when I go back to the text I'll try to keep in mind scope and audience and would appreciate thoughts (on TSV or generally) ...